site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -96,7 +96,7 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.metrics.stackdriverLabels`                       | A set of default labels to add to all stackdriver metrics generated in form of key value pair (`key=value,key2=value2`). By default metadata are automatically added using Kubernetes API and GCP metadata enpoint.     | \`\`                               |
 | `agones.metrics.serviceMonitor.interval`                 | Default scraping interval for ServiceMonitor                                                                                                                                                                            | `30s`                              |
 | `agones.serviceaccount.controller`                       | Service account name for the controller. **Note**: Will be replaced with `agones.serviceaccount.controller.name` in Agones 1.16                                                                                         | `agones-controller`                |
-| `agones.serviceaccount.sdk`                              | Service account name for the sdk. **Note**: Will be replaced with `agones.serviceaccount.sdk.name` in Agones 1.16                                                                                                       | `agones-sdk`                       | 
+| `agones.serviceaccount.sdk`                              | Service account name for the sdk. **Note**: Will be replaced with `agones.serviceaccount.sdk.name` in Agones 1.16                                                                                                       | `agones-sdk`                       |
```

Comment:
Not related to your PR, but was this field already replaced since Agones is already way pass 1.16. Same for the line 98.

Reply:
Looking into the code, the replacement was already done so I think it should be fine to remove these two lines.